lib/ffmpeg/timeout.rb

```diff
@@ -16,6 +16,7 @@ def pause
 
     def resume
       @paused = false
+      tick
```

Comment:
this one seems to cause a race condition, I would change the order of the lines

the `tick` call is setting the `@last_tick` to the current date to not cause a timeout after a long pause, but the loop int the other thread can do its check between setting `@paused` to false and setting the `@last_tick` to the current time. this could end up breaking the sleep loop while we meant to keep it running (hence setting the time to be `Time.now`)